tests/unit/ingestion/validators/sourceValidator.spec.ts

```diff
@@ -23,9 +25,9 @@ describe('SourceValidator', () => {
 
   beforeEach(() => {
     config = container.resolve<IConfig>(SERVICES.CONFIG);
-    mockGdalInfoValidator = { validateInfoData: jest.fn } as unknown as GdalInfoValidator;
+    mockGdalInfoManager = { getInfoData: jest.fn, validateInfoData: jest.fn } as unknown as GdalInfoManager;
     mockGpkgManager = { validateGpkgFiles: jest.fn } as unknown as GpkgManager;
-    sourceValidator = new SourceValidator(jsLogger({ enabled: false }), config, mockGdalInfoValidator, mockGpkgManager);
+    sourceValidator = new SourceValidator(jsLogger({ enabled: false }), config, mockGdalInfoManager, mockGpkgManager);
```

Comment:
line 43 why do we use "unsupportedCrs"?

Reply:
removed

Reply:
changed to "anotherValidInputFiles"

Reply:
in general it does not matter because we only checks that all files exist, in this current situation we does not care (yet) about other validations